keps/sig-apps/4017-pod-index-label/kep.yaml

```diff
@@ -12,18 +12,18 @@ approvers:
   - "@soltysh"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.28"
+latest-milestone: "v1.32"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha:
   beta: "v1.28"
-  stable:
+  stable: "v1.32"
 
 feature-gates:
```

Comment:
One more nit, can you copy the metrics from the SLI question from PRR at the bottom of this document, in the `metrics` section as well.

Reply:
done, PTAL